agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -3329,14 +3329,20 @@ bool backhaul_manager::handle_1905_topology_discovery(const std::string &src_mac
         return false;
     }
 
+    uint32_t if_index   = message_com::get_uds_header(cmdu_rx)->if_index;
+    std::string if_name = network_utils::linux_get_iface_name(if_index);
+    if (if_name.empty()) {
+        return false;
```

Comment:
So log the error here, where you can explicitly say it's in the topology discovery handling.

Reply:
Done